gui/slick/views/layouts/main.mako

```diff
@@ -48,14 +48,18 @@
     </head>
     <body ${('data-controller="' + controller + '" data-action="' + action + '" api-key="' + sickbeard.API_KEY +'"  api-root="api/v2/"', '')[title == 'Login']}>
         <%include file="/partials/header.mako"/>
-        % if submenu:
-        <%include file="/partials/submenu.mako"/>
-        % endif
-        <%include file="/partials/alerts.mako"/>
+        
         <div id="contentWrapper">
+        <div class="row">
```

Comment:
Was playing with aligning the submenu and the main content. As the main content always has a left and right margin. While I think the submenu should also have this.